BistroQ.Presentation/Views/UserControls/Client/ProductDetailControl.xaml

```diff
@@ -0,0 +1,125 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="BistroQ.Presentation.Views.UserControls.Client.ProductDetailControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:BistroQ.Presentation.Views.UserControls.Client"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:converters="using:BistroQ.Presentation.Converters"
+    xmlns:lvc="using:LiveChartsCore.SkiaSharpView.WinUI"
+    mc:Ignorable="d">
+
+    <UserControl.Resources>
+        <converters:NumberToCurrencyConverter x:Key="NumberToCurreny"/>
+        <converters:BooleanToVisibilityConverter x:Key="BoolToVisibility"/>
+        <converters:ReverseBooleanConverter x:Key="ReverseBool"/>
+    </UserControl.Resources>
+
+    <ScrollView Width="480">
```

Comment:
Add `Dragging` gesture for this `ScrollView`

Reply:
Added.